src/acmp/acmp.c

```diff
@@ -0,0 +1,9 @@
+#include "em_acmp.h"
+
+ACMP_TypeDef* GET_ACMP0() {
```

Comment:
Move to file `get_acmp.c`, my argument: these are auxiliary functions added to get the pointer reference. Lets keep them separate from the STATIC_INLINE issue. Down the line I think we should move all `GET_<>` function into one file. Thoughts?